tests/drone/setup-server-and-app.sh

```diff
@@ -22,12 +22,11 @@ else
 	then
 		php occ config:system:set web.baseUrl --value="http://owncloud/index.php/apps/web"
 		php occ config:system:set enable_previews --type=boolean --value=false
+		php occ config:system:set skeletondirectory --value=''
```

Comment:
Don't set the skeleton folder in the core so that the same test can be run for both core and ocis